seqr/views/utils/export_utils.py

```diff
@@ -96,17 +99,19 @@ def export_multiple_files(files, zip_filename, file_format='csv', add_header_pre
             for filename, header, rows in files:
                 header_display = header
                 if add_header_prefix:
-                    header_display = map(
-                        lambda (i, k): '{}-{}'.format(str(i).zfill(2), k), enumerate(header)
-                    )
+                    header_display = ['{}-{}'.format(str(i_k[0]).zfill(2), i_k[1]) for i_k in enumerate(header)]
```

Comment:
`i_k` is a weird variable name, maybe do something descriptive like `indexed_column`

Reply:
Okay. Change it to `index_value`.